vulcan/_message.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+from datetime import datetime
+
+from related import (
+	TimeField,
+    IntegerField,
+    StringField,
+    DateField,
+    ChildField,
+    immutable,
+    to_model,
+)
+
+from ._subject import Subject
+from ._teacher import Teacher
+from ._utils import sort_and_filter_date
+@immutable
+class Message:
+	id = IntegerField(key="WiadomoscId")
+	sender_id = StringField(key="NadawcaId")
+	recipients = StringField(key="Adresaci")
+	title = StringField(key="Tytul")
+	content = StringField(key="Tresc")
+	sent_data = StringField(key="DataWyslania")
+	sent_time = StringField(key="GodzinaWyslania")
+	folder= StringField(key="FolderWiadomosci")
+	sender = ChildField(Teacher, required=False)
+	@classmethod
+	def get(cls, api, date=None):
+		if not date:
+			date=api.student.period.from_
```

Comment:
You could make the function `get` to have 2 optional parameters: `date_from` and `date_to`, and of course if they're `None`, use period dates.